### PR TITLE
update ci version and fix publish error

### PR DESCRIPTION
**Pull request by kevindiu**
This PR update the CI version and also fix the publish error.

Reference of fixing the public error:
https://github.com/storybookjs/storybook/issues/15#issuecomment-353965422

Signed-off-by: kevindiu <kevindiujp@gmail.com>

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -83,11 +83,11 @@ jobs:
     steps:
       - name: Install Protoc
         uses: arduino/setup-protoc@v1
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: '10'
-          registry-url: 'https://registry.npmjs.org'
-      - uses: actions/checkout@v1
+          registry-url: 'https://registry.npmjs.org/'
```
Comment (vankichi) [inline .github/workflows/release.yml:89]: `registry-url` is must needed?

Comment (kevindiu) [reply .github/workflows/release.yml:89]: It is optional field, I am sorry that I couldn't find any document indicating the default value of this field.

But seems the official document set this value without `/` in the end, so the publish issue may cause by another mistake.
https://github.com/actions/setup-node/blob/d08cf222111d5c1d21b3cd4b958937f818d10d9a/docs/advanced-usage.md

Comment (vankichi) [reply .github/workflows/release.yml:89]: I see.
btw, can you publish packages which have been failed to publish?

Comment (vankichi) [reply .github/workflows/release.yml:89]: And, please sign CLA :)

Comment (vankichi) [reply .github/workflows/release.yml:89]: How is going?

**Pull request merged**